examples/MITRotor_quickstart.ipynb

```diff
@@ -0,0 +1,367 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are 5 total momentum models to the user to choose from, all of which are concrete subtypes of the `MomentumModel` class. The options are: `ConstantInduction`, `ClassicalMomentum`, `HeckMomentum`, `UnifiedMomentum`, and `MadsenMomentum`. Each of these different momentum models requires different arguments. These are as follows:\n",
+    "\n",
+    "- `ConstantInduction`: argument `a`, which is the constant induction for the rotor. The default is $\\frac{1}{3}$.\n",
+    "\n",
+    "- `ClassicalMomentum`: no arguments.\n",
+    "\n",
+    "- `HeckMomentum`: arguments `averaging` and `ac`. The options for `averaging` are: `\"sector\"`, `\"annulus\"`, or `\"rotor\"` (with a default of `\"rotor\"`). The default value of `ac`, the initial induction value, is $\\frac{1}{3}$. \n",
+    "\n",
+    "- `UnifiedMomentum`: arguments `averaging` and `beta`. The argument `averaging` has the same options as for the `HeckMomentum` model. The argument `beta` is the shear layer growth rate parameter and has a default value of 0.1403.\n",
+    "\n",
+    "- `MadsenMomentum`: argument `cosine_exponent`, which is an optional exponent on the cosine term in the Madsen model.\n",
+    "\n",
+    "The `HeckMomentum` model with default arguments is provided to the `BEM` as the momentum model if the user doesn't provide one.\n",
+    "\n",
+    "Here we show an example of the `UnifiedMomentum` model:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "unified_momentum_model = mr.UnifiedMomentum(averaging = \"rotor\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Tangential Induction Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are two options for the tangential induction model, `NoTangentialInduction` and `DefaultTangentialInduction`, which are concrete subtypes of the `TangentialInductionModel` class. Neither takes any arguments, and as the name suggests, the `DefaultTangentialInduction` is the default model provided to the `BEM`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tangential_induction_model = mr.DefaultTangentialInduction()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Aerodynamic Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are two options for aerodynamic models, `KraghAerodynamics` and `DefaultAerodynamics`, both of which are concrete subtypes of the `AerodynamicModel` class. They do not take any arguments, and are called internally to get the needed `AerodynamicProperties` for our calculations. As the name suggests, `DefaultAerodynamics` is the default for the BEM unless otherwise specified."
```

Comment:
Reference https://onlinelibrary.wiley.com/doi/full/10.1002/we.1612 for `KraghAerodynamics'